backend/pkg/database/package.go

```diff
@@ -121,6 +123,29 @@ func (p *PackageTableHandler) Create(pkg *Package) error {
 	return nil
 }
 
+func (p *PackageTableHandler) Delete(pkg *Package) error {
+	if err := p.packagesTable.Delete(pkg).Error; err != nil {
+		return fmt.Errorf("failed to delete package: %v", err)
+	}
+
+	p.viewRefreshHandler.TableChanged(packageTableName)
+
+	return nil
+}
+
+func (p *PackageTableHandler) GetDBPackage(id string) (*Package, error) {
```

Comment:
similar to `GetDBResource(id string, shouldGetRelationships bool)` we don't want/need to get the package relationships?